packages/utils/src/bits.cairo

```diff
@@ -0,0 +1,37 @@
+use crate::bit_shifts::shr;
+
+pub fn bit_len(n: u64) -> u64 {
```

Comment:
There should be `BitSize` impl for u64 https://github.com/starkware-libs/cairo/blob/932767340c4b8a762140bf9eba305f437587ac1b/corelib/src/test/num_test.cairo#L10

Reply:
If I understand correctly, `BitSize` returns size of the type, but `bit_len` returns size of the value, i.e. the number of bits after trimming left zeros

Reply:
For instance, `bit_len(5)` is `3`, because `5` is `0b_101`.